### PR TITLE
Data Science - Added Data Analytics Resources

### LINKED ISSUE #334: 💡 [FEATURE] - <Adding a section for Data Analytics>
### Idea Contribution

- [X] I have read all the feature request issues.
- [X] I'm interested in working on this issue
- [X] I'm part of GSSOC organization

### Explain feature request

Data Analytics is one of the most demanding skill in the present job market, after the likes of AI/ML. So its important to have a section dedicated to Data Analytics, helping all the aspiring Data Analysts

### Explain your solution

Being an aspiring Data Analyst myself, i can provide useful study materials, course links(YouTube channels) and resources for learning this domain and its application.

### Any alternative approaches/features

_No response_

### Additional Context

_No response_

### PR DESCRIPTION
**Pull request by MIpaltan007**
### Closes Issue

Closes #334 

### Description: 

Added A Dedicated  YouTube Playlist for Data Analytics along with an Introduction to this domain

### Resource URL: 

https://www.youtube.com/@AlexTheAnalyst/community
https://youtu.be/yZvFH7B6gKI?si=tOFgcis4V2V737sc

### Checklist:
Before submitting your pull request, ensure that you have completed the following tasks: 

- [x] I have carefully reviewed and adhered to the [contributing guidelines](https://github.com/jfmartinz/ResourceHub/blob/main/CONTRIBUTING.md) before creating this pull request.
- [x] I followed the prescribed PR title template. _(Check this if you're adding a resource)_

### CODE REVIEW

File: Data Science/readme.md [hunk 1]
```diff
@@ -152,6 +152,10 @@ Data science is the field that utilizes scientific methods, algorithms, and syst
         <td> <a href="https://youtu.be/X2vAabgKiuM?si=ppqTTm_mui7aOtdT">Natural Language Processing and Recommender systems </a></td>
         <td>Branch of AI enabling computers to understand, interpret, and generate human language, vital for text analysis tasks and algorithms suggesting items based on user preferences, leveraging data to enhance personalized recommendations in various domains.</td>
       </tr>
+      <tr>
+      <td><a href="https://youtu.be/yZvFH7B6gKI?si=hYj4x1sAGk9TK1VY">Data Analytics</a></td>
+      <td>Data analytics converts raw data into actionable insights. It includes a range of tools, technologies, and processes used to find trends and solve problems by using data. Data analytics can shape business processes, improve decision-making, and foster business growth.</td>
+      </tr>
```
Comment (jfmartinz) [inline Data Science/readme.md:158]: This is a youtube video about 'What is Data Science'. This resource should not be included here. You can remove it or find another tutorial or courses about data sceience and move it to the Free courses or Paid courses if its free or not free

**Review by jfmartinz — APPROVED** [review]

File: Data Science/readme.md (hunk 1 above)
Comment (jfmartinz) [inline Data Science/readme.md:158]: This is a youtube video about 'What is Data Science'. This resource should not be included here. You can remove it or find another tutorial or courses about data sceience and move it to the Free courses or Paid courses if its free or not free

**Review by jfmartinz — CHANGES_REQUESTED** [review]

File: Data Science/readme.md (hunk 1 above)
Comment (MIpaltan007) [reply Data Science/readme.md]: The changes are done. You can see the changes and review them

**Comment by MIpaltan007** [comment]
@jfmartinz Please have a look now

**Review by jfmartinz — APPROVED** [review]
Looks good @MIpaltan007, thanks!

**Pull request merged**